Podfile

```diff
@@ -175,7 +175,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.80.0'
+  gutenberg commit: '6094df206c82b139b14634c05ec5b963ddba3f97'
```

Comment:
Friendly reminder: we'll need to update this reference to a newly created alpha tag before merging.